src/Makefile.am

```diff
@@ -123,6 +123,7 @@ libdmtcpinternal_a_SOURCES = coordinatorapi.cpp 		\
 			     procselfmaps.cpp			\
```

Comment:
In the commit message for `Use WrapperLock in dlsym to avoid deadlock.`, you write:

> Here is the stacktrace of forkexec user-thread that gets into deadlock
while the ckpt-thread has started acquiring locks in acquireLock():
>
> ...

I think you forgot to fill in the `...`.  Could you add the missing stacktrace?

Reply:
I added the following stacktrace but git removed it because of the leading `#`. Fixed now :).
``` User Thread:
(gdb) bt
...
#6  futex_wait (...) at ../include/futex.h:21
#7  DmtcpMutexLock (...) at mutex.cpp:47
#8  dmtcp::ThreadSync::delayCheckpointsLock () at threadsync.cpp:269
#9  dmtcp_disable_ckpt () at dmtcpplugin.cpp:132
#10 dmtcp_dlsym (...) at dmtcp_dlsym.cpp:553
#11 realloc (...) at alloc/mallocwrappers.cpp:82
#12 __add_to_environ (...) at setenv.c:154
#13 __setenv (...) at setenv.c:259
#14 getUpdatedLdPreload (...) at execwrappers.cpp:506
#15 patchUserEnv (...) at execwrappers.cpp:616
#16 dmtcp_execvpe (...) at execwrappers.cpp:870
#17 0x000055d06cf6d37d in main (...) at forkexec.c:55

CKPT Thread
(gdb) bt
...
#6  futex_wait (...) at ../include/futex.h:21
#7  DmtcpMutexLock (...) at mutex.cpp:47
#8  DmtcpRWLockWrLock (...) at rwlock.cpp:94
#9  dmtcp::ThreadSync::acquireLocks () at threadsync.cpp:156
#10 dmtcp::DmtcpWorker::waitForCheckpointRequest () at dmtcpworker.cpp:426
#11 checkpointhread (...) at threadlist.cpp:420
...```